docs/REST API/rest-api.md

```diff
@@ -8979,49 +8986,156 @@ Request (using `filters`):
-| batch_execution_id | string | path | **Required**. The ID returned from a batch script run. |
+| team_id         | integer | query | _Available in Fleet Premium_. Filters to batch script runs for the specified team. |
+| status          | string  | query | Filters to batch script runs with this status. Either `"started"`, `"scheduled"`, or "`completed`". |
+| page            | integer | query | Page number of the results to fetch. |
+| per_page        | integer | query | Results per page. |
 
 
 #### Example
 
-`GET /api/v1/fleet/scripts/batch/summary/abc-def`
+`GET /api/v1/fleet/scripts/batch`
 
 ##### Request body
 
 ```json
 {
-  "batch_execution_id": "e797d6c6-3aae-11ee-be56-0242ac120002"
+  "team_id": 123,
+  "status": "completed"
+}
+```
+
+##### Default response
+
+`Status: 200`
+
+```json
+{
+  "batch_executions": [
+    {
+      "script_id": 555,
+      "script_name": "my-script.sh",
+      "team_id": 123,
+      "not_before": "2025-07-01T15:00:00Z",
+      "completed_at": "2025-07-06T15:00:00Z",
+      "status": "completed",
+      "canceled": false,
+      "targeted_host_count": 12599,
+      "ran_host_count": 12345,
+      "pending_host_count": 234,
+      "errored_host_count": 18,
+      "incompatible_host_count": 3,
+      "canceled_host_count": 2
+    }
+  ],
+  "meta": {
+    "has_next_results": false,
+    "has_previous_results": false
+  }
 }
 ```
 
+### Get batch script
+
+Returns a summary of a batch-run script, including host counts and current status.
+
+> The [Get batch script summary](https://github.com/fleetdm/fleet/blob/fleet-v4.71.1/docs/REST%20API/rest-api.md#get-batch-script-summary) endpoint is deprecated as of Fleet 4.73. It is maintained for backwards compatibility. Please use this endpoint instead.
+
+`GET /api/v1/fleet/scripts/batch/:batch_execution_id`
+
+#### Parameters
+
+| Name               | Type    | In   | Description                                 |
+| ------------------ | ------- | ---- | --------------------------------------------|
+| batch_execution_id | string  | path | **Required**. The ID returned from a batch script run. |
+
+
+#### Example
+
+`GET /api/v1/fleet/scripts/batch/abc-def`
+
+
 ##### Default response
 
 `Status: 200`
 
 
 ```json
 {
-  "ran": 12345,
-  "pending": 234,
-  "errored": 18,
-  "canceled": 2,
-  "targeted": 12599,
   "script_id": 555,
   "script_name": "my-script.sh",
-  "team_id": 123
+  "team_id": 123,
+  "not_before": "2025-07-01T15:00:00Z",
+  "completed_at": "2025-07-06T15:00:00Z",
+  "status": "completed",
+  "canceled": false,
+  "targeted_host_count": 12599,
+  "ran_host_count": 12345,
+  "pending_host_count": 234,
+  "errored_host_count": 18,
+  "incompatible_host_count": 3,
+  "canceled_host_count": 2,
 }
 ```
 
+### List hosts targeted in batch script
+
+Returns a list hosts targeted in a batch script run, along with their script execution status.
+
+`GET /api/v1/fleet/scripts/batch/:batch_execution_id/results`
```

Comment:
```suggestion
`GET /api/v1/fleet/scripts/batch/:batch_execution_id/hosts`
```

Reply:
I could make a case for either.  The table this comes from is `batch_activity_host_results` 😄 .